subsys/bluetooth/mesh/pb_gatt_srv.c

```diff
@@ -44,7 +44,9 @@
 	 BT_LE_ADV_OPT_ONE_TIME | ADV_OPT_USE_IDENTITY |                       \
 	 ADV_OPT_USE_NAME)
 
-static bool prov_fast_adv;
+#define FAST_ADV_TIME (60ll * MSEC_PER_SEC)
```

Comment:
Minor (and sorry for not pointing this out earlier): if you need the explicit "long long" annotation (I'm not convinced that you do, but I'll assume you've checked this), use upper-case LL instead of lower-case ll. Lower case looks too much like 11 (depending on your font) and is not consistent with how this is done elsewhere in the tree (e.g. unsigned uses U and not u)

Reply:
This is intentional usage. Otherwise, there will be some Coverity warning about the mistyped comparison.